tests/smoke-tests/test-balances-consistency.ts

```diff
@@ -523,6 +523,7 @@ describeSmokeSuite("S300", `Verifying balances consistency`, (context, testIt) =
   });
 
   testIt("C200", `should match total supply`, async function () {
+    this.timeout(30000);
```

Comment:
The test does only 1 request, not sure 30s is needed

Reply:
The time needed is to reduce across the entire set of accounts in a data blob, which seems to be sometimes more than a few seconds on current hardware.